### PR TITLE
add python-executor package

### PR DESCRIPTION
**Pull request by rpetit3**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Hi conda-forge team!

I used `conda skeleton pypi executor` to create it and cleaned up/added info where needed.

Please let me know if there are any corrections I should make or if there are 'best practices' I failed to follow.

Thank you very much for your time and work!

Cheers,
Robert

### CODE REVIEW

**Comment by rpetit3** [comment]
I think I submitted this PR too early. A dependency `property-manager` was just recently merged (https://github.com/conda-forge/staged-recipes/pull/9442). I'll wait till its available to retry the checks.

File: recipes/executor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "executor" %}
+{% set version = "21.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "ea8d4e914186d52e3902763be25bd3ae531cd720f0f07e7a211d76f2d6a326cc"
```
Comment (marcelotrevisani) [inline recipes/executor/meta.yaml]:
```suggestion
  sha256: ea8d4e914186d52e3902763be25bd3ae531cd720f0f07e7a211d76f2d6a326cc
```
Could you please remove the ``"`` from the ``sha256`` ?

**Comment by marcelotrevisani** [comment]
We are having some problem with the bot which creates the feedstock, it may take some hours to your package ``property-manager`` to be available. Sorry about that!

**Comment by rpetit3** [comment]
Looks like property-manager (https://github.com/conda-forge/property-manager-feedstock) is live now. 

```
conda install -c conda-forge property-manager
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/rpetit3/miniconda3

  added / updated specs:
    - property-manager


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    humanfriendly-4.18         |           py37_0         103 KB  conda-forge
    property-manager-2.3.1     |             py_0          22 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         125 KB

The following NEW packages will be INSTALLED:

  humanfriendly      conda-forge/linux-64::humanfriendly-4.18-py37_0
  property-manager   conda-forge/noarch::property-manager-2.3.1-py_0
  verboselogs        conda-forge/noarch::verboselogs-1.7-py_0

The following packages will be UPDATED:

  ca-certificates    pkgs/main::ca-certificates-2019.5.15-1 --> conda-forge::ca-certificates-2019.6.16-hecc5488_0

The following packages will be SUPERSEDED by a higher-priority channel:

  certifi                                         pkgs/main --> conda-forge
  conda                                           pkgs/main --> conda-forge
  openssl              pkgs/main::openssl-1.1.1c-h7b6447c_1 --> conda-forge::openssl-1.1.1c-h516909a_0


Proceed ([y]/n)? n


CondaSystemExit: Exiting.

```

Is there a way for me to retrigger the checks? Or should I wait. Thanks for all the help!

**Comment by scopatz** [comment]
I have triggered a rebuild

**Comment by rpetit3** [comment]
Awesome! Thank you very much @scopatz

**Pull request merged**